.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,21 @@
+name: PR
+
+on:
+  pull_request:
+    branches:
+      - master
+
```

Comment:
Should this specify master as the branch for pull requests?

Reply:
I need to ask Ian about this one, I followed his code in validator repo but the schemas one doesn't specify master in branches.

Reply:
I believe this will only trigger the actions on pull requests on Master; we shouldn't restrict the running of the actions in that way. Very rarely do we pull into branches other than master, but when we do we should trigger this workflow.

Reply:
ok, removing now👍